en/docs/install-and-setup/upgrading-the-solution/reporting-data-migration.md

```diff
@@ -36,11 +36,14 @@ as follows:
     ```
    
 5. Replace the existing `deployment.yaml` file in the Streaming Integrator as follows:
-   - Go to the `<SI_HOME>/<OB_BI_ACCELERATOR_HOME>/repository/resources` directory.
-   - Rename `wso2si-4.0.0-deployment.yaml` to `deployment.yaml`.
-   - Copy the `deployment.yaml` file to the `<SI_HOME>/conf/server` directory to replace the existing file.
+    - Go to the `<SI_HOME>/<OB_BI_ACCELERATOR_HOME>/repository/resources` directory.
+    - Rename `wso2si-4.0.0-deployment.yaml` to `deployment.yaml`.
+    - Copy the `deployment.yaml` file to the `<SI_HOME>/conf/server` directory to replace the existing file.
+
+6. Open the `<SI_HOME>/conf/server/deployment.toml` file, and configure the hostnames and databases related
+   properties accordingly.
```

Comment:
Let's mention to point the databases creates for Open Banking 2.0.

Reply:
Fixed in f9ac8569d3756f0ee97e73443ce1c7ea47c0aa5b